### ISSUE TITLE
tink-worker: error logged without actionStatus

### ISSUE THREAD

**Issue by rgl** [tinkerbell/tink#502]
The following tink-worker error is logged without actionStatus:

https://github.com/tinkerbell/tink/blob/836244b4ae687821f088673cbd0acd1169695227/cmd/tink-worker/internal/worker.go#L318-L319

Maybe this needs to be changed to:

```go
l.With("actionStatus", actionStatus.ActionStatus.String()).Error(err) 
```

Or to also pass `l` context to `exitWithGrpcError`, this needs to be changed to:

```go
l := l.With("actionStatus", actionStatus.ActionStatus.String())
l.Error(err) 
```

**Comment by displague** [comment]
I've opened a PR with a fix for this (#517).  I chose the `l = l.With` approach because `l` is passed into another function in a conditional branch.

Thanks for opening the issue and identifying the fixes!

**Comment by mmlb** [comment]
#517 is merged, this should have been closed then.
